roles/shiftstack/defaults/main.yml

```diff
@@ -34,4 +34,5 @@ cifmw_shiftstack_qa_repo: "https://review.gerrithub.io/shiftstack/shiftstack-qa"
 cifmw_shiftstack_run_playbook: "ocp_testing.yaml"
 cifmw_shiftstack_sc: "local-storage"
 cifmw_shiftstack_shiftstackclient_artifacts_dir: "/home/cloud-admin/artifacts"
+cifmw_shiftstack_exclude_artifacts_regex: "openshift-install"
```

Comment:
Hey Ramón, even if it's just a cosmetic thing, we're following an alphabetical order for these vars definition, can you please follow the same approach?
Can you please also add the param to [ci-framework/roles/shiftstack/README.md](https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/shiftstack/README.md)?

Reply:
Thanks for your comments! Fixed